src/site/content/en/blog/strict-csp/index.md

```diff
@@ -408,29 +422,25 @@ calculate CSP hashes for inline scripts using this
 [tool](https://strict-csp-codelab.glitch.me/csp_sha256_util.html).
 {% endAside %}
 
-{% Details %}
-
-{% DetailsSummary %}
-About `async = false` and script loading `async = false`
-isn't blocking in this case, but use this with care.
-{% endDetailsSummary %}
+#### Script loading considerations
```

Comment:
Most of the other changes in this PR are cosmetic/standardize formatting, but this is the one structural change. Instead of nesting `{% Details %}` elements here, I just took the inner `{% Details %}` and included the text under the heading `Script loading considerations`.